Bottom: f3f911c611d16ced1ca75d16730b4a663b07787f
Top:    e6ace61a1378f7f00cdc9e49bae497a658f6271c
Author: Parth Shah <parth@linux.ibm.com>
Date:   2020-01-10 17:26:58 +0530

Refresh of sched-fair-tune-task-wake-up

---

diff --git a/kernel/sched/fair.c b/kernel/sched/fair.c
index ed0df4abe874..a5cee6876263 100644
--- a/kernel/sched/fair.c
+++ b/kernel/sched/fair.c
@@ -5888,7 +5888,7 @@ static int select_idle_cpu(struct task_struct *p, struct sched_domain *sd, int t
 #ifdef CONFIG_SCHED_SMT
 
 /* Define non-idle CPU/task as the one with the utilization >= 12.5% */
-#define is_zealous(util) ((util) > (100 >> 3))
+#define is_zealous(util) ((util) > (1024 >> 3))
 
 static inline bool is_background_task(struct task_struct *p)
 {
